source/Coin.hx

```diff
@@ -0,0 +1,40 @@
+package;
+
+/**
+ * ...
+ * @author Samuel Faulkner
+ */
+
+import flixel.util.FlxColor;
+import flixel.FlxSprite;
+ 
+class Coin
+{
+	/**
+	 * 
+	 * @param	X	X-coordinate of the coin
+	 * @param	Y	Y-coordinate of the coin
+	 * @param	red	Bool value that tells creator if coin is red or yellow
+	 * @return
+	 */
+	static public function createCoin(X:Float=0, Y:Float=0, red:Bool=false):FlxSprite 
+	{
```

Comment:
See comment below. Also, was there a particular reason that you use a static factory-type method (https://stackify.com/static-factory-methods/) like this as opposed to defining "new"?